src/pages/_app.js

```diff
@@ -44,26 +46,28 @@ export default class App extends NextApp {
 
     return (
       <Container>
-        <JssProvider
```

Comment:
This looks worse than it is. All I did was put `<ComponentsProvider value={componentsContext}>` around `JssProvider`